admin/js/boldgrid-backup-admin-home.js

```diff
@@ -25,10 +25,10 @@ BOLDGRID.BACKUP.HOME = function( $ ) {
 
 	// Onload event listener.
 	$( function() {
-		var $urlImportSection = $( '#url-import-section' );
+		var $urlImportSection = $( 'div#url-import-section' );
 
 		// On click action for the Upload button.
-		$( '#upload-archive-form' )
+		$( 'div#url-import-section' )
```

Comment:
You just assigned `$urlImportSection` above? Why not just use that variable here?

Reply:
Probably cause I made that change before I changed the definition above :D. It's been adjusted